src/components/collect_run_display/collect_run_display.gd

```diff
@@ -199,7 +202,7 @@ func _on_ErrorBtn_pressed():
 
 
 func start_test():
-	var path := "res://data/test_data/websocket_workflow_collect_run.txt"
```

Comment:
do we need both of the websocket_workflow_collect_run.txt or we should replace that with the websocket_workflow_collect_run_2.txt. Shouldnt we rename websocket_workflow_collect_run_2.txt to websocket_workflow_collect_run.txt ?